test/orchard/inspect_test.clj

```diff
@@ -319,3 +472,58 @@
              (:path (-> inspector (inspect/down 0)))))
       (is (= '[:a (nth 2) :b :c (nth 73) (find :foo) key class <unknown>]
              (:path (-> inspector (inspect/down 0) (inspect/down 1))))))))
+
+;; Datafy and Nav
+
+(defn- skip-to-datafy-section [result]
+  (drop-while #(not= "Datafy: " %) result))
+
+(when datafy?
+  (deftest inspect-class-datafy-test
+    (testing "inspecting a class renders the datafy section"
+      (is (= inspect-result-datafy-class
+             (-> (class (Object.))
+                 inspect
+                 render
+                 skip-to-datafy-section))))))
+
+(when datafy?
+  (deftest inspect-ref-datafy-test
+    (testing "inspecting a ref renders the datafy section"
+      (is (= inspect-result-datafy-ref
+             (-> (atom {:a 1})
+                 inspect
+                 render))))))
+
+(when datafy?
+  (deftest inspect-namespace-datafy-test
+    (testing "inspecting a namespace renders the datafy section"
+      (is (= inspect-result-datafy-namespace
+             (-> (create-ns 'orchard.inspect-test-ns)
+                 inspect
+                 render))))))
+
+(when datafy?
+  (deftest inspect-metadata-extension-datafy-test
+    (testing "inspecting a map extended with `p/datafy` renders the datafy section"
+      (is (= inspect-result-datafy-metadata-extension
+             (-> (with-meta {:name "John Doe"}
+                   {'clojure.core.protocols/datafy (fn [x] (assoc x :type 'Person))})
+                 inspect render skip-to-datafy-section))))))
+
+(when datafy?
+  (deftest inspect-nav-extension-datafy-test
+    (testing "inspecting a map extended with `p/nav` renders the datafy section"
+      (is (= inspect-result-datafy-nav-extension
+             (-> (with-meta {:name "John Doe"}
+                   {'clojure.core.protocols/datafy
+                    (fn [x] (assoc x :type 'Person))
+                    'clojure.core.protocols/nav
+                    (fn [coll k v]
+                      [k (get coll k v)])})
+                 inspect render skip-to-datafy-section))))))
+
+(when datafy?
+  (deftest inspect-throwable-datafy-test
+    (testing "inspecting a throwable renders the datafy section"
+      (is (seq (-> (ex-info "BOOM" {}) inspect render skip-to-datafy-section))))))
```

Comment:
Looks like the `(seq` assertion could be more self-evident - we cannot tell if it really means what `testing` says it means :)

Reply:
Fixed